Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM sealink/phpdocker:7.0.12
 
 # Set craft version
-ENV CRAFTURL 'https://download.craftcdn.com/craft/2.6/2.6.2957/Craft-2.6.2957.zip'
+ENV CRAFTURL 'https://download.craftcdn.com/craft/2.6/2.6.2958/Craft-2.6.2958.zip'
```

Comment:
Next time we do this can we extract this further, e.g.:
```
ENV CRAFTURL 'https://download.craftcdn.com/craft/2.6/2.6.2958/Craft-2.6.2958.zip'
```
becomes
```
ENV CRAFTPATCH=2958
ENV CRAFTURL 'https://download.craftcdn.com/craft/2.6/2.6.$CRAFTPATCH/Craft-2.6.$CRAFTPATCH.zip'
```

Just to make all your updates every simpler and easier!